### PR TITLE
This is a test feature...

### PR DESCRIPTION
**Pull request by jonnylynchy**
This adds an html file to the repo.

### CODE REVIEW

**Comment by qpedrick** [comment]
Needs comments

**Review by jonnylynchy — COMMENTED** [review]
This looks great, I don't know why Q denied this.

**Comment by jonnylynchy** [comment]
> Needs comments

This has comments. What do you mean?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TEsty testerson</title>
```
Comment (jonnylynchy) [inline index.html]: This title has a typo. I think you meant to do this?
```<title>Testy Testerson</title>```

**Review by qpedrick — COMMENTED** [review]
Great work!

**Review by qpedrick — APPROVED** [review]

**Pull request merged**